### PR TITLE
Support for Symfony 4

### PR DESCRIPTION
**Pull request by Nyholm**
#SymfonyConHackday2017

### CODE REVIEW

**Comment by gianarb** [comment]
Thanks a lot!

**Comment by Nyholm** [comment]
Travis file is way better now.. But I cannot find the syntax error.

File: .travis.yml [hunk 1]
```diff
@@ -1,14 +1,61 @@
...
+        - php: 7.2
+          env: COVERAGE=true PHPUNIT_FLAGS="-v --coverage-text"
+
+          # Test LTS versions. This makes sure we do not use Symfony packages with version greater
+          # than 2 or 3 respectively. Read more at https://github.com/symfony/lts
+        - php: 7.2
+          env: DEPENDENCIES="dunglas/symfony-lock:^2"
+        - php: 7.2
+          env: DEPENDENCIES="dunglas/symfony-lock:^3"
+         - php: 7.2
```
Comment (joelwurtz) [inline .travis.yml]: extra space here, causing the syntax error pooke @Nyholm

**Comment by Nyholm** [comment]
Thank you Joel. 

Tests are finally green now. (Im pretty sure, they have not been running at the time of writing)
@gianarb, let me know if I can do anything more with this PR.

**Comment by gianarb** [comment]
Thanks a lot

**Pull request merged**